README.md

```diff
@@ -43,8 +43,29 @@ You must also have Java installed, and available on your PATH as `java`.
 
 ### Desktop
 
-If you are building for a Linux platform, make sure that the GTK 3 development packages are
-installed on your system. (Ubuntu: `libgtk-3-dev`, Fedora: `gtk3-devel`)
+If you are building for a Linux platform, the following are typical dependencies:
+<h3>Ubuntu</h3>
+
+* Java (OpenJDK 11 will suffice)
+* Libgtk-3-dev
+* Libssl-dev
+* Libasound2-dev
+* \*gobject\*
```

Comment:
Is there any special meaning for the surrounding `*`?

Reply:
There are enough packages within that particular group of dependencies that I feel it's just more expedient to wildcard the entire term.

Reply:
Probably a wildcard, as there isn't just one package called gobject. That said, we definitely shouldn't recommend to install them manually, right? They aren't explicitly needed on Actions workers after all.

Reply:
Perhaps not, but my compilations have always errored without them.